database/seed-data/fakeDataGenerator.js

```diff
@@ -1,9 +1,9 @@
 const faker = require('faker');
 
-let fakeHeader = () => (faker.lorem.sentence().slice(0, -1));
-let fakeSentence = () => (faker.lorem.sentence());
-let fakeDescription = () => (faker.lorem.sentences());
-let fakeTitle = () => (faker.name.jobArea());
+let fakeHeader = () => faker.lorem.sentence().slice(0, -1);
```

Comment:
Are you trying to get part of the sentence as a header?

Reply:
This has been refactored to one line function expressions. I am using the sentence generator and removing the period.